### PR TITLE
Changed backend redirect address to new server

### PR DESCRIPTION
**Pull request by fharding1**

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -1,3 +1,3 @@
-/api/*  https://pigmice-scouting.herokuapp.com/:splat 200!
+/api/*  http://162.243.156.241/:splat 200!
```
Comment (calebeby) [inline _redirects]: Can you use https?

**Comment by fharding1** [comment]
No.

**Comment by calebeby** [comment]
Why not?

**Comment by calebeby** [comment]
Can't you add a let's encrypt certificate?

**Comment by fharding1** [comment]
We can't use letsencrypt. We can't use cloudflare unless we get a domain
which you said we shouldn't do.

**Comment by calebeby** [comment]
Why not?

**Comment by calebeby** [comment]
Why do we need cloudflare for let's encrypt?

**Comment by fharding1** [comment]
Letsencrypt provides an SSL cert. Cloudflare provides SSL as a sort of man in the middle, meaning data between your client and cloudflare is encrypted, but not cloudflare and your server. We can't install let's encrypt on our container AFAIK unless docker cloud has an option which I can't check right now since I'm not home and the site isn't rendering correctly on my phone. We can't use cloudflare since we don't have a domain.

**Comment by calebeby** [comment]
Right now:
```
user -> netlify CDN -> docker cloud
                    ^
```
I don't like that that link is not secure.

With cloudflare:
```
user -> netlify CDN -> CloudFlare -> docker cloud
                                  ^
```
There is still an unsecured link.

**Comment by calebeby** [comment]
Seems like it might be possible: https://forums.docker.com/t/custom-ssl-to-nginx-not-working/37754/4

**Comment by BrendanBurkhart** [comment]
I thought we were hosting on digital ocean or whatever, shouldn’t be able to install whatever we want on there?

**Comment by fharding1** [comment]
In theory yeah but if you even wanna deploy to a different droplet or to a
different server it's not gonna have SSL and you're gonna have to
completely reconfigure it.

**Comment by calebeby** [comment]
But put it in your dockerfile

**Comment by fharding1** [comment]
How are we gonna keep the SSL cert across reboots?

**Comment by fharding1** [comment]
With a domain (api.pigmice.ga) I was able to add HTTPS. HTTPS is being setup in the golang app, not with an nginx reverse proxy or cloudflare. You use docker to map your certificates from the host to the container (ex. /etc/ssl/fullchain.pem:/etc/letsencrypt/live/api.pigmice.ga/fullchain.pem).

**Pull request merged**